inspirehep/modules/theme/templates/inspirehep_theme/format/record/Job_HTML_detailed.tpl

```diff
@@ -20,142 +20,212 @@
+            </div>
+          {% endif %}
+
+          {% if record.continent %}
+            <div class="detailed-jobs-information">
+              Region:
+              {{ record.continent | join(', ') }}
+            </div>
+          {% endif %}
+
+          {% if record.urls %}
+            <div class="detailed-jobs-information">
+              More information:
+              {% for url in record.urls %}
+                <a href="{{ url }}">
+                  {{ url }}
+                </a>
+              {% endfor %}
+            </div>
+          {% endif %}
+
+          {% if record.creation_modification_date %}
+            {% if record.creation_modification_date[0] %}
+              <div class="detailed-jobs-information">
+                Date added: {{ record.creation_modification_date[0].creation_date | format_date }}
+              </div>
+            {% endif %}
+          {% endif %}
+        </div>
+
+        <div class="detailed-jobs-information" id="job-remove-this-listing">
+          <a class="btn custom-btn btn-warning pdf-btn no-external-icon" href="mailto:jobs@inspirehep.net?subject=Remove-listing-{% if record['acquisition_source'] %}{{ record['acquisition_source'][0]['submission_number']}} {% endif %}&body=Please remove listing {% if record['acquisition_source'] %}{{ record['acquisition_source'][0]['submission_number']}} {% endif %} https://inspirehep.net/jobs/{{ record['control_number'] }}/edit" role="button" title="Remove this listing">Remove this listing</a>
+        </div>
+      </div>
+
+      <div class="col-md-3" id="record-detailed-jobs-deadline">
+        {% if record.deadline_date %}
+          <div class="detailed-jobs-information">
+            <div class="alert alert-info" id="detailed-jobs-deadline-alert" role="alert">
+              Deadline: {{ record.deadline_date | format_date }}
+            </div>
+          </div>
         {% endif %}
-      {% endfor %}
-      </h3>
-    </div>
-  </div>
-  <div class="row">
-    <div class="col-md-12">
-      <h4 class="pull-left">
-        <b>
-          {{ record['position'] }}
-        </b>
-        {% if record['institution'][0]['name']%}
-          {% if record['continent'] %}
-            (<a href="/search?p=department_acronym:{{ record['institution'][0]['name'] }}&cc=Institutions">{{ record['institution'][0]['name'] }}</a> - {{ record['continent'] }})
-          {% else %}
-            (<a href="/search?p=department_acronym:{{ record['institution'][0]['name'] }}&cc=Institutions">{{ record['institution'][0]['name'] }}</a>)
+
+        {% if record.institution[0].name %}
+          <div id="job-detailed-map">
+            <script>
+              function initMap() {
+                var mapDiv = document.getElementById('job-detailed-map');
+                var address = '{{ record.institution[0].name }}';
+                var geocoder = new google.maps.Geocoder();
+                var map = new google.maps.Map(mapDiv, {
```

Comment:
I have this weird map visualization bug:
<img width="985" alt="weird-map-bug" src="https://cloud.githubusercontent.com/assets/381280/15176194/3207c81c-176a-11e6-821f-9e14b7d0d8da.png">

(This is record 1408251)